apps/web/src/state/predictions/index.ts

```diff
@@ -170,7 +171,7 @@ export const fetchNodeHistory = createAsyncThunk<
   const { bufferSeconds } = getState()
 
   // Turn the data from the node into a Bet object that comes from the graph
-  const bets: Bet[] = roundData.reduce((accum, round) => {
+  const bets: Bet[] = roundData.reduce((accum: any | PredictionsRoundsResponse, round: PredictionsRoundsResponse) => {
```

Comment:
the type of `accum` should be `Bet[]`?

Reply:
I'm try to set it to `Bet[]` but still throw error

Reply:
hmm what is the error exactly? Or let's simply set any to bypass